docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  isaac-sim:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      args:
+        USER_ID: ${USER_ID:?}
+        GROUP_ID: ${GROUP_ID:?}
+        SHARED_GROUP_ID: ${SHARED_GROUP_ID}
+        DOCKER_USERNAME: ${DOCKER_USERNAME:?}
+    image: ${USERNAME:?}/nvsynth
+    user: ${USER_ID:?}:${GROUP_ID:?}
+    volumes:
+      - .:/nvsynth
+      # food data volume
+      - ${DATASET_PATH:-/dev/null}:/food
```

Comment:
Actually nvm, creating a symlink inside the container is a bad idea since the link will still point to a location inside the container once it exists (which breaks the dir on the host filesystem). Instead, let's change this so that it mounts it directly into the right directory
```suggestion
      - ${DATASET_PATH:-/dev/null}:/assets/food
```